packages/client/hmi-client/src/services/models/simulation-service.ts

```diff
@@ -32,11 +26,59 @@ export async function getRunStatus(runId: number) {
 
 export async function getRunResult(runId: number) {
 	try {
-		const resp = await API.get(`simulation-service/${runId}/result`);
+		const resp = await API.get(`simulation/${runId}/result`);
 		const output = resp.data;
 		return output;
 	} catch (err) {
 		logger.error(err);
 		return null;
 	}
 }
+
+export async function getSimulation(id) {
```

Comment:
Let's leverage the Types as much as possible

```suggestion
export async function getSimulation(id: Simulation["id"]): Simulation {
```